README.md

```diff
@@ -15,6 +15,7 @@ Chewy is an ODM and wrapper for [the official Elasticsearch client](https://gith
 * [Why Chewy?](#why-chewy)
 * [Usage](#usage)
   * [Client settings](#client-settings)
+    * [AWS ElasticSearch configuration] (#aws-elastic-search)
```

Comment:
Hey @pyromaniac, there's an extra white-space here, which is messing up the Markdown styling.
It should be:
`    * [AWS ElasticSearch configuration](#aws-elastic-search)`